### PR TITLE
ASP.NET Core - Platform - SingleQuery

### PR DESCRIPTION
**Pull request by benaadams**
/cc @sebastienros

### CODE REVIEW

File: frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs [hunk 1]
```diff
@@ -63,6 +64,10 @@ public void OnStartLine(HttpMethod method, HttpVersion version, Span<byte> targe
                 {
                     requestType = RequestType.Fortunes;
                 }
+                else if (Paths.SingleQuery.Length <= path.Length && path.StartsWith(Paths.SingleQuery))
```
Comment (sebastienros) [inline frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs]: Should we order these length tests from the shortest to the biggest paths?

Comment (benaadams) [reply frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs]: Done

Comment (fafhrd91) [reply frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs]: Do you guys see performance difference compared to simple string comparison?

Comment (benaadams) [reply frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs]: The Length test is a simple memory test, the StartsWith is a non-inlining call - it probably won't be a huge factor, but why make the call for everything its not?

**Pull request merged**